content/en/docs/spin-operator/tutorials/scaling-with-hpa.md

```diff
@@ -11,44 +11,71 @@ Horizontal scaling, in the Kubernetes sense, means deploying more pods to meet d
+Run the following command to create a Kubernetes cluster that has [the containerd-wasm-shims](https://github.com/deislabs/containerd-wasm-shims) pre-requisites installed: If you have a Kubernetes cluster already, please feel free to use it:
 
 ```console
-git clone https://github.com/spinkube/spin-operator.git
-cd spin-operator
+k3d cluster create wasm-cluster-scale \
+  --image ghcr.io/spinkube/containerd-shim-spin/k3d:v0.12.0 \
+  -p "8081:80@loadbalancer" \
+  --agents 2
 ```
 
-## Setting Up Kubernetes Cluster
+### Deploying Spin Operator and its dependencies
 
-Run the following command to create a Kubernetes cluster that has [the containerd-wasm-shims](https://github.com/deislabs/containerd-wasm-shims) pre-requisites installed: If you have a Kubernetes cluster already, please feel free to use it:
+First, you have to install [cert-manager](https://github.com/cert-manager/cert-manager) to automatically provision and manage TLS certificates (used by Spin Operator's admission webhook system). For detailed installation instructions see [the cert-manager documentation](https://cert-manager.io/docs/installation/).
 
 ```console
-k3d cluster create wasm-cluster-scale --image ghcr.io/spinkube/containerd-shim-spin/k3d:v0.12.0 -p "8081:80@loadbalancer" --agents 2
+# Install cert-manager CRDs
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.3/cert-manager.crds.yaml
+
+# Add and update Jetstack repository
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+
+# Install the cert-manager Helm chart
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.14.3
 ```
 
-Next, from within the `spin-operator` directory, run the following commands to install the Spin runtime class and Spin Operator:
+Next, run the following commands to install the Spin [Runtime Class]({{<ref "glossary#runtime-class" >}}) and Spin Operator [Custom Resource Definitions (CRDs)]({{<ref "glossary#custom-resource-definition-crd">}}):
+
+> Note: In a production cluster you likely want to customize the Runtime Class with a `nodeSelector` that matches nodes that have the shim installed. However, in the K3d example, they're installed on every node. 
 
 ```console
-kubectl apply -f config/samples/spin-runtime-class.yaml
-make install
+# Install the RuntimeClass
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.0.2/spin-operator.runtime-class.yaml
+
+# Install the CRDs
+kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.0.2/spin-operator.crds.yaml
 ```
 
-Lastly, start the operator with the following command:
+Lastly, install Spin Operator using `helm` and the [shim executor]({{< ref "glossary#spin-app-executor-crd" >}}) with the following commands:
 
 ```console
-make run
+# Install Spin Operator
+helm upgrade spin-operator \
+  --namespace spin-operator \
```

Comment:
We'd need a `--create-namespace` arg as well, if we don't otherwise instruct the user to create it separately.